### PR TITLE
FIX: issue with bundler renaming class constructor

### PR DESCRIPTION
**Pull request by NeoPlays**

### CODE REVIEW

File: launcher/src/backend/ethereum-services/GethService.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export class GethService extends NodeService {
 
         const service = new GethService();
         service.init(null,
+            "GethService",
```
Comment (stefa2k) [inline launcher/src/backend/ethereum-services/GethService.js:12]: shouldn't there be another `null` as argument for the id?

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ethereum-services/GethService.js (hunk 1 above)
Comment (stefa2k) [reply launcher/src/backend/ethereum-services/GethService.js:12]: or switch `null` with `"GethService"` one line up?

**Review by maxbehzadi82 — DISMISSED** [review]
lg

File: launcher/src/backend/ethereum-services/GethService.js (hunk 1 above)
Comment (NeoPlays) [reply launcher/src/backend/ethereum-services/GethService.js:12]: ohhh man and i even tripple checked 😅

**Review by stefa2k — APPROVED** [review]
thx

**Review by stefa2k — APPROVED** [review]

**Pull request merged**